### PR TITLE
Bootstrap project

### PR DESCRIPTION
**Pull request by Jhonatanssal**
This is my solution to the seventh Microverse project.

It has been built using technologies like:

-HTML.

-Bootstrap.

CSS:
- Flexbox
- Grid

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -0,0 +1,573 @@
...
+}
+
+.sticky-navbar {
+  position: sticky;
+  top: 0;
+  width: 100%;
+  left: 0;
+  z-index: 9;
+  height: 5rem !important;
+}
```
Comment (bertil291utn) [inline styles/style.css:53]: ## No `!important` overusing[1]

The !important rule was conscripted in the mid-to-late 90’s to help web designers and developers override the rules of their style sheet. 

- [ ] Remove all `!important` rules in your entire CSS file

<hr>

**Note:** Check all this entire page and fix them

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

**Review by bertil291utn — CHANGES_REQUESTED** [review]
# **Status:** Changes requested 🟠 
<p align="center"> image <br><strong>Not yet</strong> </p>

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it GitHub Flow you have a professional README file. 

However, in order to accomplish as the actual page, you required to do these changes, listed below:
### General requirements

- [x] Check linter errors [\[more\]](/microverse/guides/tse/how_to_be_a_tse/-/blob/master/code_review_checklist/04_linters.md) as requested [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/)

* Make sure that students use config files copied from [config linters repo](https://github.com/microverseinc/linters-config#linters-config)

  - [x] Therefore please add the linters directory 

- [x] There's extra padding in the entire page, please remove

image

- [x] Media queries for small screen should be from 0 to 768px, please fix that approach. Check this [video](https://www.loom.com/share/3e42a811e1534835b5ee768b9712696f)

### Specific req.

| Section/Screen                             | Large | Medium | Small |
| :----------------------------------------- | :---: | :----: | :---: |
| Navbar              |   1   |      |       |
| Featured stories              |   3   |        |       |
| Top story             |       |        |       |
| My turn             |       |        |       |
| Cultural&Travel             |      |        |       |
| Opinion                 |       |        |       |
| Latest news                  |       |        |       |
| More stories |   2   |        |       |
| In the magazine         |   4    |    6    |   8    |
| Editor's pick |   5 |     |      |
| Footer                                     |       |    7    |       |
###### This table is given each section with his screen width approach, each issue is given by number below this table

1. Just remove extra navbar it's not required

image

2. The image should have `position: relative` to add text under the image, check the original approach

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
|   
image
              |   
image
    |

3. Add the tag on each article in this section

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| 
image
  |
image |

4.  In the magazine, the title should align centered

image

5. Add tags in the editors pick articles section 

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| 
image
|
image |

6. Add in the magazine's title
image

7.  Remove extra padding that footer has. Same approach for large and small screens

image

8. Add in the magazine title
image


### OPTIONAL

- [x] You might use of the [`<footer>`](https://www.w3schools.com/TagS/tag_footer.asp) tag for the footer

### Summary
- You have small approach, however the media query is not right 
-  Work on the small screen approach 
  - Large min-width: 1024px up
  - Medium min-width: 768px up
  - Small up to 768px
-  Don't forget to work with inline comments below this comment box.

***

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

File: styles/style.css (hunk 1 above)
Comment (Jhonatanssal) [reply styles/style.css:53]: Thank yo very much for your review, I already made the changes you requested for.

Happy coding.

**Review by peterrobert — CHANGES_REQUESTED** [review]
### **CHANGES REQUIRED** :recycle: :stop_sign: 
### HELLO @jssarrazolaa  :wave: YOU HAVE DONE A GOOD JOB ON THIS PROJECT. GREAT WORK IMPROVING YOUR PROJECT LIKE REQUESTED. HOWEVER, THERE ARE A FEW MORE CHANGES TO BE DONE KINDLY CHECK BELOW :arrow_down: 

- [x] At the mobile view, there is an overflow at the x-axis. Kindly you need to fix that.
over-flow

- [x] At tablet view some of your wording on the subscribe section are being hidden kindly, add some padding between the image and the wordings
tablet view

- [x] Also as you can see below :arrow_down:  some of the elements in the newsletter section are falling of the container. Try adding some padding 
off container

### [OPTIONAL]
In the desktop view if you would love to add an amination just like in the original website. That is in the feature stories, top story and the debate, you can use the code below in your css. Try using
  position: sticky;
    top: 0;
    z-index: 0;


### IF YOU HAVE ANY QUESTION CONCERNING THE REVIEW REACH ME VIA SLACK @PETERROBERT

### HAPPY CODING

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
# Status: Changes requested :red_circle: 

## Hello @jssarrazolaa :wave: 

## You have done an amazing job fixing the last required changes, congratulations! :clap: 

## However, we still need to make some changes:

- [x] At the mobile size, there's still an overflow in the X axis (check the horizontal bar):

Screenshot from 2020-08-13 17-07-20
<br>

- [x] In the Subscribe section (Medium screen size), the words are going over the images:
Screenshot from 2020-08-13 17-12-22
<br>

- [x] Your live link is not working in your README file, kindly change the link.

- [ ] (**OPTIONAL**) In the main three columns section, try aligning both the top and bottom section, so we don't have any empty spaces:
## Yours:
Screenshot from 2020-08-13 17-15-06
<br>
## Original:
Screenshot from 2020-08-13 16-33-08
<br>

### And that's it, please make the changes and submit for another review :pray: 

### If you have any doubts, please talk to me on Slack! :handshake: 

### Happy coding!

**Review by kevcap — APPROVED** [review]
# [STATUS] PROJECT APPROVED 🍾

  

### Hey @jssarrazolaa, congrats, you did an excellent job making the previously requested changes 💪!

  

_**Great attention to details, nice readme, and the design a looks great, I got my eyes on you buddy, keep up the good work!!**_ 🥇

  

You can now merge these changes. I wish the very best and see you at the top🚀

  

  

<hr>

  

👨🏻‍💻 **TSE: Kevin Alves**

  

  

📫 Have any question? Better call Kevin:

  

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

**Pull request merged**